public/app/features/explore/utils/links.ts

```diff
@@ -60,3 +61,23 @@ function getTitleFromHref(href: string): string {
   }
   return title;
 }
+
+function buildHostUrl() {
+  return `${window.location.protocol}//${window.location.host}${config.appSubUrl}`;
+}
+
+function getRelativeURLPath(url: string) {
+  let path = url.replace(buildHostUrl(), '');
+  return path.startsWith('/') ? path.substring(1, path.length) : path;
+}
+
+export const createShortLink = memoizeOne(async function(path: string) {
```

Comment:
memoized createShortLink.

Reply:
Did we implement any auto cleanup for the links? Like would this prevent refreshing some last used counter or something?